content/blog/2024/09/camel48-whatsnew/index.md

```diff
@@ -0,0 +1,193 @@
+---
+title: "Apache Camel 4.8 What's New"
+date: 2024-09-16
+authors: [davsclaus, squakez, orpiske, oscerd]
+categories: ["Releases"]
+preview: Details of what we have done in the Camel 4.8 release.
+---
+
+Apache Camel 4.8 LTS has just been [released](/blog/2024/09/RELEASE-4.8.0/).
+
+This release introduces a set of new features and noticeable improvements that we will cover in this blog post.
+
+## Camel Core
+
+The simple language has new functions such as a `iif` (ternary if).
+The `@BindToRegistry` now supports init/destroy methods, and can be declared as lazy as well.
+
+The `log` EIP can more easily configure its logger name using dynamic patterns.
+
+Add `poll` EIP as an easier and simpler version of `pollEnrich` EIP which is also more tooling friendly.
+
+The Camel tracer has been greatly performance improved, and reducing overhead.
```

Comment:
The performance of the Camel tracer has been significantly improved, with a reduction of overhead.

Reply:
Thanks